### ISSUE TITLE
cmrt_dump.txt generated 

### ISSUE THREAD

**Issue by zjin-lcf** [intel/llvm#5379]
I find that cmrt_dump.txt is generated after running a SYCL program with CUDA support. The file is empty.

**Comment by AlexeySachkov** [comment]
@zjin-lcf, am I right the request here is to explain why that file is being generated and moreover, avoid generating it, right?

**Comment by zjin-lcf** [comment]
Yes. You are right. Thanks.

**Comment by romanovvlad** [comment]
@zjin-lcf
Could you please provide all the information asked in the template so we can analyze the issue.
Template:

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Please describe the steps to reproduce the behavior:

1. Include code snippet as short as possible
2. Specify the command which should be used to compile the program
3. Specify the comment which should be used to launch the program
4. Indicate what is wrong and what was expected

**Environment (please complete the following information):**

- OS: [e.g Windows/Linux]
- Target device and vendor: [e.g. Intel GPU]
- DPC++ version: [e.g. commit hash or output of `clang++ --version`]
- Dependencies version: [e.g. low-level runtime versions (like NEO 20.04)]

**Additional context**
Add any other context about the problem here.

**Comment by zjin-lcf** [comment]
Sorry. Could the developers of CM explain when this file is generated ?

**Comment by bader** [comment]
I bet it was caused by ESIMD emulator, which build was enabled for a few days. @dongkyunahn-intel, does it seem like a possible cause?
@zjin-lcf, could you make a clean build of a new compiler and check if this file is still generated, please?

@AlexeySachkov, I don't think it's related to CUDA.

**Comment by zjin-lcf** [comment]
Yes, I built the compiler with CUDA support today. The file is generated.  Can you reproduce that ?

**Comment by dongkyunahn-intel** [comment]
`cmrt_dump.txt` is generated by CM - https://github.com/intel/cm-cpu-emulation/blob/3c9eb58b06b72d28de3015d23243f0fa594cdb31/libcmrt/cm_statistics.cpp#L95.

However, ESIMD_EMULATOR using CM is enabled only with `--enable-esimd-emulator` option in origin/sycl branch (https://github.com/intel/llvm/pull/5366). I would suggest pulling up-to-date `origin/sycl` branch to their own working branch and doing clean build after cleaning their `$BUILD` directory. (`rm -rf $BUILD/*`)

I did clean build using origin/sycl without ESIMD_EMULATOR support. I cannot find any file related to CM in `$BUILD/install` directory.

**Comment by zjin-lcf** [comment]
I didn't delete the 'build' directory.  Thank you for your suggestions!

**Comment by lsatanov** [comment]
> @zjin-lcf, am I right the request here is to explain why that file is being generated and moreover, avoid generating it, right?

Hi. I'll get rid of it in the next CM EMU opensource release.

**Comment by zjin-lcf** [comment]
Okay.

**Comment by lsatanov** [comment]
Release PR that fixes this:
https://github.com/intel-innersource/drivers.gpu.compiler.cmc.emu.staging/pull/130
